assets/src/components/map.tsx

```diff
@@ -106,11 +107,33 @@ const Map = (props: Props): ReactElement<HTMLDivElement> => {
   const stops = (props.shapes || []).flatMap((shape) => shape.stops || [])
   const tileType = props.tileType || "base"
 
+  useEffect(() => {
+    const classList = mapRef.current?.getContainer().classList
+    if (props.extraClasses && classList) {
+      for (const [className, shouldBePresent] of Object.entries(
+        props.extraClasses
+      )) {
+        if (shouldBePresent && !classList.contains(className)) {
+          classList.add(className)
+        }
+        if (!shouldBePresent && classList.contains(className)) {
+          classList.remove(className)
+        }
+      }
```

Comment:
```suggestion
      Object.entries(props.extraClasses)
        .map(([className, shouldBePresent]) => 
          classList.toggle(className, shouldBePresent))
```

Reply:
That does look like a much simpler way of doing that - thanks!